base/applications/mspaint/mouse.cpp

```diff
@@ -630,6 +772,7 @@ struct TextTool : ToolBase
         textEditWindow.ValidateEditRect(&rc);
         textEditWindow.ShowWindow(SW_SHOWNOACTIVATE);
         textEditWindow.SetFocus();
+        updateLast(x, y);
```

Comment:
Why is it called only after all the SetWindowText etc. etc. SetFocus calls?

Reply:
Removed.